README.md

```diff
@@ -28,17 +28,21 @@ To use the D rules, add the following to your `WORKSPACE` file to add the
 external repositories for the D toolchain:
 
 ```python
-http_archive(
+load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
+git_repository(
```

Comment:
You can actually use http_archive with a fixed commit:
https://github.com/bazelbuild/rules_d/archive/bcf137e3c9381545ce54715632bc1d31c51ee4da.tar.gz